benchmark/single-source/DataBenchmarks.swift

```diff
@@ -168,17 +168,30 @@ public let DataBenchmarks = [
     runFunction: { string($0*200, from: largeUnicodeData) }, tags: d,
     legacyFactor: 50),
 
-  BenchmarkInfo(name: "StringToDataEmpty",
-    runFunction: { data($0*200, from: emptyString) }, tags: d,
+  BenchmarkInfo(name: "StringToDataFromUTF8ViewEmpty",
```

Comment:
New tests look reasonable to me. I have in the past been requested to not change these unit test names, though. /CC @palimondo for comment

Reply:
Thanks Itai. We will now have two "StringToData" benches so I thought it was worth naming them, but we could keep the old name if that's required.

Reply:
I completely agree — I think the tests can be renamed. From what I remember, there will be a path forward on renaming these tests while maintaining a mapping to the existing names. The new tests can likely use the new [Benchmark Naming Conventions](https://github.com/apple/swift/blob/master/benchmark/Naming.md) and the old tests will be renamed later (there's some discussion back in #21766 about this)

Reply:
Yeah, leave the existing benchmarks unchanged and name the new ones like this:
* `String.data.Empty`
* `String.data.Small`
* `String.data.Medium`

Reply:
There will be another pass in the future to clean up the names in DataBenchmarks to new naming convention, while maintaining long-term performance tracking.